### DISCUSSION TITLE
Can I auto update my GitHub rep

### DISCUSSION THREAD

**Discussion by nafiulhaq** [community/community#129662] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Are
Question 

### Body

Can I auto update my GitHub rep from my PC. Is that possible .. for example I create a rep and clone them but after that I change that folder and it auto change here is that possible?

**Comment by nafiulhaqueinan** [comment]
no its not possible you must commit changes

**Comment by ebndev** [comment]
Hi @nafiulhaqueinan, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
